packages/react-checkbox/src/components/Checkbox/useCheckboxStyles.ts

```diff
@@ -4,226 +4,209 @@ import { CheckboxState } from './Checkbox.types';
 
 export const checkboxClassName = 'fui-Checkbox';
 
+// TODO replace these spacing constants with theme values once they're on the theme
+const spacingHorizontalS = '8px';
+const spacingHorizontalM = '12px';
+
+// CSS variables for the indicator colors.
+// These let the colors be picked based on the :hover, etc. states on the root
+const indicatorColor = '--fui-Checkbox-indicator-color';
```

Comment:
I wonder if it makes more sense to include the `var()` wrapper in the const? I've seen that in other places and it makes the usage cleaner.

Somewhat separate from this PR, we should have a consistent documented naming convention for variable prefixes and casing (I've seen other places use lowercase, `--fui-checkbox`).

Reply:
This constant can't include `var()` because it's used to set the value:
`[varName]: 'value',`

I chose the prefix to match the checkbox class name `fui-Checkbox`. I think they should match each other..? But maybe we need an RFC about naming css vars.

Reply:
I could go either way with the casing. We probably need an RFC or something documenting which one to use.